doc/content/reference/configuration/gateway-configuration-server.md

```diff
@@ -3,3 +3,27 @@ title: "Gateway Configuration Server Options"
 description: ""
 weight: 9
 ---
+
+## Security Options
+
+- `gcs.require-auth`: Require authentication for the HTTP endpoints
+
+## Basic Station CUPS Options
+
+The `gcs.basic-station` options configure the GCS to handle Basic Station CUPS requests.
+
+- `gcs.basic-station.allow-cups-uri-update`: Allow CUPS URI updates (default "false")
```

Comment:
Well I used this as reference:
https://github.com/TheThingsNetwork/lorawan-stack/blob/b32e468b430f17e4c9faa405a11f95e3c386720e/doc/content/reference/configuration/application-server.md#mqtt-options

Reply:
See the description of https://github.com/TheThingsNetwork/lorawan-stack/pull/1481 for how the config reference was initially approved (by you actually)

Reply:
Yeah sure but the closest reference for the GS options is the AS options. I'm not particularly against having the defaults, but since I approved the reference, I'll remove the defaults here.